sdk/src/main/java/com/bugsnag/android/ThreadState.java

```diff
@@ -15,34 +15,26 @@
 class ThreadState implements JsonStream.Streamable {
     private static final String THREAD_TYPE = "android";
 
-    final Configuration config;
+    private final Configuration config;
     private final Thread[] threads;
     private final Map<Thread, StackTraceElement[]> stackTraces;
+    private final long currentThreadId;
 
     ThreadState(Configuration config) {
         this.config = config;
         stackTraces = Thread.getAllStackTraces();
-        threads = sanitiseThreads(Thread.currentThread().getId(), stackTraces);
+        currentThreadId = Thread.currentThread().getId();
+        threads = sanitiseThreads(stackTraces);
     }
 
     /**
      * Returns an array of threads excluding the current thread, sorted by thread id
      *
-     * @param currentThreadId the current thread id
-     * @param liveThreads     all live threads
+     * @param liveThreads all live threads
      */
-    private Thread[] sanitiseThreads(long currentThreadId,
-                                     Map<Thread, StackTraceElement[]> liveThreads) {
+    private Thread[] sanitiseThreads(Map<Thread, StackTraceElement[]> liveThreads) {
```

Comment:
The doc comment (and possibly method name) should be updated to reflect the current behavior.